### ISSUE TITLE
NSMBDS romhacks sometimes black screen

### ISSUE THREAD

**Issue by poudink** [ahezard/nds-bootstrap#463]
I have had NSMBDS romhacks blackscreen while loading certain areas. For example, "New Something Mario Something" black screen on 1-2 and "Adventure Super Mario Bros." black screens on 1-2's bonus area. I've also had another romhack (forgot the name) black screen on 1-1. I can confirm that it isn't the romhacks' fault in any of these cases because I tested on emulators to make sure it worked. I am using the latest version of nds-bootstrap and twilight menu on an O3DS.

**Comment by YYoshi241** [comment]
Adding to this list:
Super Mario Endless Earth
(https://nsmbhd.net/thread/2877-super-mario-endless-earth-release/)

Crashes in the first level (and even states this in the Instructions.)

Doesn't matter the settings or if ROM is trimmed. (Yes, I tried turning SOUND FIX OFF)

Also using o3DS latest TWiLightMenu and NDS-bootstrap.

**Comment by YYoshi241** [comment]
Hey, @poudink  have you tried the latest PreRelease Bootstrap (12.3.2 PR1)?

**Comment by poudink** [comment]
I will try it soon, when I have time. I will give the results tomorrow. I'll test to see if the romhacks other than endless earth have been fixed, as I am not 100% sure the issue with endless earth was the same as with the other romhacks.

**Comment by YYoshi241** [comment]
Actually, Release 12.4 just came out and has the same fix. You should try that instead.

**Comment by poudink** [comment]
that's what I was gonna try, and that's what I tried. Now that I tested, here's what I found, with the areas that black screen/used to black screen in parantheses.
Adventure Super Mario Bros.: Still broken (1-2 bonus area)
New Something Mario Something: Fixed (1-2)
Super Mario Bros. Challenge LVLS: Still broken (1-1)
Super Mario Endless Earth: Fixed
So from what I got Endless Earth (and New Something Mario Something, apparently) black screened because the areas were too big for nds-bootstrap, right? I can assure you that's not the problem with the hacks that still black screen. The 1-2 bonus area in Adventure Super Mario Bros. that black screens is a pretty small area.
